que.go

```diff
@@ -148,7 +148,14 @@ var ErrMissingType = errors.New("job type must be specified")
 
 // Enqueue adds a job to the queue.
 func (c *Client) Enqueue(j *Job) error {
-	return execEnqueue(j, c.pool)
+	return execEnqueue(j, c.pool, "que_insert_job")
+}
+
+// ConditionalEnqueue adds a job to the queue, and checks for conflicts on the condition.
+// This condition must be included as a unique constraint on the queue table, so if you are using this,
+// you have extra plumbing to do.
+func (c *Client) ConditionalEnqueue(j *Job, cond string) error {
+	return execEnqueue(j, c.pool, "que_cond_insert_job", cond)
```

Comment:
I don't love this, but I wanted to keep the stuff thats more of an `analytics_etl` thing - like job fields/types in `analytics_etl`.